docs/modules/ROOT/pages/vshn-managed/postgresql/create.adoc

```diff
@@ -25,26 +25,32 @@ spec:
     size: <5>
       cpu: "600m"
       memory: "3500Mi"
+      requests: <6>
+        cpu: "300m"
+        memory: "1000Mi"
       disk: "80Gi"
     scheduling:
       nodeSelector:
-        appuio.io/node-class: "plus" <6>
+        appuio.io/node-class: "plus" <7>
   writeConnectionSecretToRef:
-    name: postgres-creds <7>
+    name: postgres-creds <8>
 ----
 <1> Instance name
 <2> The namespace where the object will be created
 <3> PostgreSQL version
 <4> Specify custom PostgreSQL settings [optional]
-<5> Cpu, memmory and disk configurations
-<6> Specify the node class for the instance [optional]
-<7> Secret where the connection details are provisioned. This secret shouldn't exist before creation.
+<5> Cpu, memory and disk configurations
+<6> CPU and memory requests configuration [optional]
+<7> Specify the node class for the instance [optional]
+<8> Secret where the connection details are provisioned. This secret shouldn't exist before creation.
 
 NOTE: To get more information about all available configuration options, please see the xref:references/crds.adoc#k8s-api-github-com-vshn-component-appcat-apis-vshn-v1-vshnpostgresql[API Reference]
 
 
 NOTE: The PostgreSQL instance will be scheduled on the platforms default node class. You can override this by setting the `spec.parameters.scheduling.nodeSelector` field accordingly. For more information see https://docs.appuio.cloud/user/references/node-classes.html[Node Classes]
 
+NOTE: By default the resource requests and limits are set to the values defined in `spec.parameters.size.cpu` and `spec.parameters.size.memory`. If you want to explicitely set resource requests that are lower than the limit you need to define them in `spec.parameters.size.requests`.
+
```

Comment:
I'd like to make this a bit more explicit that we don't recommend doing this for production instance

Reply:
Refactored that section and added a notice